docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,611 @@
+// @flow strict
+import { type Node } from 'react';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+
+export default function DropdownShowCase(): Node {
+  return (
+    <Page title="Dropdown - Show case">
+      <PageHeader badge="pilot" name="Dropdown" type="guidelines" />
+      <MainSection name="Examples">
+        <MainSection.Subsection>
+          <MainSection.Card
+            cardSize="lg"
+            title="More items on list then viewport supports"
+            description="This case shows you a list of items and your viewport has less available height then your dropdown list requests."
+            defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const [selectedDropdown, setSelectedDropdown] = React.useState(null);
+  const anchorRef = React.useRef(null);
+
+  const filteredVariants = new Array(100).fill({ value: 'test', status: 'active' });
+
+  const openDropdown = () => setOpen(true);
+  const closeDropdown = () => setOpen(false);
+
+  // DOM structure based on error: https://pinterest.slack.com/archives/C13KLG5P0/p1658953735281499
+  return (
+    <React.Fragment>
+      <Box width={360}>
+        <TapArea onTap={openDropdown} ref={anchorRef}>
```

Comment:
This should toggle the dropdown open and closed, not just open it — clicking on the button when the dropdown is open should close the dropdown